tests/test_standard_coinswap.rs

```diff
@@ -28,6 +29,9 @@ fn create_wallet_and_import(rpc: &Client, filename: PathBuf) -> Wallet {
     let mnemonic =
         mnemonic::Mnemonic::new_random(bitcoin_wallet::account::MasterKeyEntropy::Sufficient)
             .unwrap();
+    if filename.exists() {
+        fs::remove_file(&filename).unwrap();
+    }
```

Comment:
ACK, but this needs a rebase on the current master.

Reply:
Noted. I think I'll change tact also, whereby the wallet files are deleted when the test ends, so they don't clutter into repo changes.